addons/web/models/ir_qweb_fields.py

```diff
@@ -45,7 +45,7 @@ def _get_src_urls(self, record, field_name, options):
             filename = options['filename']
         else:
             filename = record.display_name
-        filename = filename.replace('/', '-').replace('\\', '-').replace('..', '--')
+        filename = (filename or 'name').replace('/', '-').replace('\\', '-').replace('..', '--')
```

Comment:
Since https://github.com/odoo/odoo/pull/81788#issuecomment-1090525079 seems to be a wanted behavior and not something that's gonna be reverted / fixed, seems good